std_msgs/Header header
int64 num_of_features
FeatureTrackerResult[] features